src/api.js

```diff
@@ -31,8 +31,27 @@ const getArticlebyArticleID = (article_id) => {
     })
 }
 
+const incrementVotes = (article_id) => {
+    return axios.patch(`https://jakes-news-api.herokuapp.com/api/articles/${article_id}`,
+    {inc_votes: 1})
+    .then((res) => {
+        return res;
+    })
+    
+}
+
+const decrementVotes = (article_id) => {
+    return axios.patch(`https://jakes-news-api.herokuapp.com/api/articles/${article_id}`,
+    {inc_votes: -1})
+    .then((res) => {
+        return res;
+    })
+}
+
```

Comment:
Your `incrementVotes` and `decrementVotes` are very similar save for the number `inc_votes`. Find a way to use only one api call for this because essentially this is just one patch request with the same endpoint.